[C++ API] torch::nn::GLU and F::glu

yf225 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #29933 [C++ API] torch::nn::FractionalMaxPool{2,3}d module and functional
* **#29922 [C++ API] torch::nn::GLU and F::glu**
* #29920 [C++ API] torch::nn::GroupNorm and F::group_norm

Differential Revision: [D18558818](https://our.internmc.facebook.com/intern/diff/D18558818)

Commit:
[C++ API] torch::nn::GLU and F::glu

Commit:
Update on "[C++ API] torch::nn::GLU and F::glu"

[C++ API] torch::nn::GLU and F::glu

gh-metadata: pytorch pytorch 29922 gh/yf225/76/head

Commit:
Update on "[C++ API] torch::nn::GLU and F::glu"

[C++ API] torch::nn::GLU and F::glu

gh-metadata: pytorch pytorch 29922 gh/yf225/76/head

Commit:
Update on "[C++ API] torch::nn::GLU and F::glu"

[C++ API] torch::nn::GLU and F::glu

gh-metadata: pytorch pytorch 29922 gh/yf225/76/head

kostmo commented:
## CircleCI build failures summary
As of commit 6cc31ac:
* **1/1** failures introduced in this PR
* **0/1** recognized as flaky



Here are the [reasons each build failed](https://dr.pytorch.org/commit-details.html?sha1=6cc31ac31a4080857dbe775196614b6706b801e5):

| Job | Step | Log excerpt |
| --- | --- | --- |
| pytorch_linux_xenial_py3_clang5_asan_test | Test | `RuntimeError: test_jit failed!` |


---
This comment was automatically generated by [Dr. CI](https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment).
Follow [this link to opt-out](https://dr.pytorch.org/admin/comments-opt-out.html) of these comments for your Pull Requests.


Please report bugs/suggestions on the [GitHub issue tracker](https://github.com/kostmo/circleci-failure-tracker/issues).

Commit:
Update on "[C++ API] torch::nn::GLU and F::glu"

[C++ API] torch::nn::GLU and F::glu

gh-metadata: pytorch pytorch 29922 gh/yf225/76/head
